docs/specifications/tests/Consistency-TP/consistency02.md

```diff
@@ -1,52 +1,77 @@
-## CONSISTENCY02: RNAME consistency
+# CONSISTENCY02: SOA RNAME consistency
 
-### Test case identifier
+## Test case identifier
 
-**CONSISTENCY02:** RNAME consistency
+**CONSISTENCY02**
 
-### Objective
+## Objective
 
 All authoritative name servers must serve the same SOA record (section
-4.2.1) of [RFC 1034](https://tools.ietf.org/html/rfc1034) for the
-tested domain. As per section 3.3.13 of [RFC 1035](https://tools.ietf.org/html/rfc1035),
+4.2.1) of [RFC 1034] for the
+tested domain. As per section 3.3.13 of [RFC 1035]
```

Comment:
Have you considered using `fmt` or a similar tool to manage the line width? `fmt` is far from perfect but I use it a lot.

Reply:
No. -- It was on purpose that I did not change the line breaks to make it easier to compare.